index.html

```diff
@@ -16,31 +16,38 @@
         <div class="logo">
             <a href="#"><h2>Bilal</h2></a>
         </div>
-        <nav class="hamBurger">
+        <nav class="Menuicon">
             <span class="material-symbols-outlined">
                 menu
                 </span>
         </nav>
+        <ul class="menu">
+            <li><a class="menu_item" href="#portfolio">portfolio</a></li>
```

Comment:
- **[OPTIONAL]** Kindly use capital letters for the "Portfolio" link to be similar to the Figma template.